src/Akka.Streams.Kafka.Tests/Integration/CommittingSpec.cs

```diff
@@ -76,7 +76,7 @@ public async Task CommittingMustEnsureUncommittedMessagesAreRedelivered()
         // some concurrent publishing
         await ProduceStrings(new TopicPartition(topic1, new Partition(0)), messages.Skip(100), ProducerSettings);
 
-        var expectedResumed = messages.Skip(committedElements.Current).ToList();
+        var expectedResumed = messages.Skip(committedElements.Current + 1).ToList();
```

Comment:
Basically, the Scala code this was all originally ported from was an exclusive ending, not inclusive. That's why we've had to manually add +1 to the end of our offset lists during assertions.